AWSSDKSwiftCLI/Tests/AWSSDKSwiftCLITests/Commands/GeneratePackageManifestTests.swift

```diff
@@ -125,7 +124,6 @@ extension GeneratePackageManifest {
             clientRuntimeVersion: clientRuntimeVersion,
             crtVersion: crtVersion,
             services: services,
-            includeProtocolTests: includeProtocolTests,
```

Comment:
Here & below I cleaned up some leftovers from moving protocol tests out of main SDK Package.swift.